Gemfile

```diff
@@ -28,9 +27,20 @@ gem 'jbuilder', '~> 2.7'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.4.2', require: false
 
+gem 'importmap-rails'
+gem 'stimulus-rails'
```

Comment:
I was thinking about using stimulus to show the file contents in a modal or sidepanel. But will probably leave that for a second PR.